common/djangoapps/track/views/segmentio.py

```diff
@@ -67,7 +65,7 @@ def segmentio_event(request):
     try:
         track_segmentio_event(request)
     except EventValidationError as err:
-        log.warning(
+        log.debug(
             'Unable to process event received from Segment: message="%s" event="%s"',
             str(err),
             request.body
```

Comment:
I'm pretty new to this part of the code base, so ignore me when I'm on the wrong track.

Looks like you're fixing this on both sides: Logging EventValidationErrors at `DEBUG` level, but then also (below) preventing (several classes of) EventValidationErrors from getting raised.  Shouldn't the validation errors that we are still logging continue to be `WARNING`s?